### PR TITLE
Add medium-editor

### PR DESCRIPTION
**Pull request by j0k3r**
https://github.com/yabwe/medium-editor

### CODE REVIEW

File: files/medium-editor/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "medium-editor",
+  "repo": "yabwe/medium-editor",
+  "files": {
+    "basePath": "dist/",
+    "include": ["js/medium-editor.min.js", "css/medium-editor.min.css", "css/themes/*.min.css"]
```
Comment (jimaek) [inline files/medium-editor/update.json]: Please add `./` before each path. This will ensure that the directories will be retained and not stripped.

Comment (j0k3r) [inline files/medium-editor/update.json]: Done. Should I squash?

File: files/medium-editor/info.ini [hunk 2]
```diff
@@ -0,0 +1,5 @@
+author = "Davi Ferreira"
+github = "https://github.com/yabwe/medium-editor"
+homepage = "https://yabwe.github.io/medium-editor/"
+description = "Medium.com WYSIWYG editor clone. Uses contenteditable API to implement a rich text solution."
+mainfile = "medium-editor.min.js"
```
Comment (jimaek) [inline files/medium-editor/info.ini]: Sorry, I missed this. It should be `js/medium-editor.min.js`

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge please

**Comment by jimaek** [comment]
@j0k3r Thank you

**Comment by j0k3r** [comment]
@jimaek Thank to you too

**Pull request closed without merging**